frontend/packages/cli/src/cli/getInputContent.ts

```diff
@@ -0,0 +1,49 @@
+import fs from 'node:fs'
+import { URL } from 'node:url'
+
+function isValidUrl(url: string): boolean {
+  try {
+    new URL(url)
+    return true
+  } catch {
+    return false
+  }
+}
```

Comment:
💭 I thought URL.canParse() could be used, but it looks like this was only supported in Node.js in v19.
We have not considered the minimum version of Node.js to be supported by the CLI, but since the LTS is v18, it would be better to support that.
So it looks good as it is 👍🏻 

ref: https://developer.mozilla.org/en-US/docs/Web/API/URL/canParse_static